pkg/apis/configuration/v1alpha1/types.go

```diff
@@ -129,7 +129,7 @@ type SelectorField struct {
 
 // TracingSpec defines distributed tracing configuration.
 type TracingSpec struct {
-	SamplingRate string `json:"samplingRate"`
+	SamplingRate float64 `json:"samplingRate"`
```

Comment:
Please don't change this, it's a backwards-incompatible change and can cause production issues.